Template.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net5</TargetFramework>
+		<PackageType>Template</PackageType>
+		<PackageVersion>1.0.0</PackageVersion>
+		<PackageId>GmodNET.Templates</PackageId>
+		<Title>GmodDotNet Templates</Title>
+		<Authors>GmodNET</Authors>
+		<Description>Project Templates for GmodDotNet module</Description>
+		<PackageTags>dotnet-new;templates</PackageTags>
+		<IncludeContentInPack>true</IncludeContentInPack>
+		<IncludeBuildOutput>false</IncludeBuildOutput>
+		<ContentTargetFolders>content</ContentTargetFolders>
+	</PropertyGroup>
+	<ItemGroup>
+		<Content Include="ProjectTemplates\**\*" Exclude="ProjectTemplates\**\bin\**;ProjectTemplates\**\obj\**" />
+    	<Compile Remove="**\*" />
```

Comment:
Do not need this if `<NoBuild>true</NoBuild>` is added.

Reply:
resolved by https://github.com/GmodNET/Templates/pull/2/commits/5820dfeee45bead39c96e3adf9658f01ca7c7376